internal/kernelversion/kernel_linux.go

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+   File copied and customized based on
+   https://github.com/moby/moby/tree/v20.10.14/profiles/seccomp/kernel_linux.go
+
+   File copied from
+   https://github.com/containerd/containerd/blob/v1.7.5/contrib/seccomp/kernelversion/kernel_linux.go
+
+   File copied from
+   https://github.com/opencontainers/runc/blob/v1.3.0/libcontainer/system/kernelversion/kernel_linux.go
+
+   (Aleksa: Maybe we should put this somewhere more useful for everyone?)
```

Comment:
I don't really like containerd's implementation, seems a tad over-engineered to me. There's a Go internal implementation which is way simpler, see https://github.com/golang/go/blob/master/src/internal/syscall/unix/kernel_version_linux.go

It returns two integers, which every programmer can compare.

If there's an interest, I can try adding it to moby/sys.

Reply:
> which every programmer can compare.

You'd be surprised with how many people I've seen screw this up.

Reply:
Opened https://github.com/moby/sys/pull/204, let's see what other maintainers think of it.